official/recommendation/ncf_keras_main.py

```diff
@@ -259,6 +260,9 @@ def run_ncf(_):
           beta_1=params["beta1"],
           beta_2=params["beta2"],
           epsilon=params["epsilon"])
+      if FLAGS.dtype == "fp16":
+        optimizer = tf.compat.v1.train.experimental.enable_mixed_precision_graph_rewrite(
```

Comment:
are we planning to add the new keras way of doing this afterwards? if yes, how will we distinguish which approach to use from the flags? what mechanism do we use in the other mdoels?

Reply:
once the tf2.0 API for fp16 is done, we will start using that. Other models use the same flag and same technique for fp16. See: official/transformer/transformer_main.py, official/r1/resnet/resnet_run_loop.py